src/app/services/theme-switcher.service.ts

```diff
@@ -10,9 +10,11 @@ import { ThemeDefinition } from "../interfaces/theme";
 export class ThemeSwitcherService {
   public readonly localStorageKey = "theme";
   private readonly fallbackTheme = "Light";
+  private preferredBrowserTheme = "";
 
   constructor() {
-    const cachedTheme = localStorage.getItem(this.localStorageKey) || this.fallbackTheme;
+    this.preferredBrowserTheme = this.getPreferredBrowserTheme === 1 ? "Light" : "Dark";
```

Comment:
I think this line can be simplified to `const cachedTheme = Theme[this.getPreferredBrowserTheme];`. This should return the enum name as string, perhaps you need to chain an additional `toString()` afterwards.